cf/data/data.py

```diff
@@ -10965,41 +10941,85 @@ def flip(self, axes=None, inplace=False, i=False):
 
         return d
 
+    @daskified(_DASKIFIED_VERBOSE)
     def HDF_chunks(self, *chunks):
-        """TODO."""
-        _HDF_chunks = self._HDF_chunks
+        """Get or set HDF chunk sizes.
 
-        if _HDF_chunks is None:
-            _HDF_chunks = {}
-        else:
-            _HDF_chunks = _HDF_chunks.copy()
+        The HDF chunk sizes may be used by external code that allows
+        `Data` objects to be written to netCDF files.
 
-        org_HDF_chunks = dict(
-            [(i, _HDF_chunks.get(axis)) for i, axis in enumerate(self._axes)]
-        )
+        Deprecated at version TODODASK and is no longer available. Use
+        the methods `nc_clear_hdf5_chunksizes`, `nc_hdf5_chunksizes`,
+        and `nc_set_hdf5_chunksizes` instead.
 
-        if not chunks:
-            return org_HDF_chunks
+        .. seealso:: `nc_clear_hdf5_chunksizes`, `nc_hdf5_chunksizes`,
+                     `nc_set_hdf5_chunksizes`
 
-        chunks = chunks[0]
+        :Parameters:
 
-        if chunks is None:
-            # Clear all chunking. Never change the value of the
-            # _HDF_chunks attribute in-place.
-            self._HDF_chunks = None
-            return org_HDF_chunks
+            chunks: `dict` or `None`, *optional*
+                Specify HDF chunk sizes.
 
-        axes = self._axes
-        for axis, size in chunks.items():
-            _HDF_chunks[axes[axis]] = size
+                When no positional argument is provided, the HDF chunk
+                sizes are unchanged
```

Comment:
```suggestion
                sizes are unchanged.
```

Reply:
... likewise for typos in deprecated methods. Someone somewhere might will be glad for our efforts!